igcollect/strongswan.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""igcollect - Strongswan stats
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import re
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='strongswan')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    template = args.prefix + '.{} {} ' + str(int(time()))
+    print(template.format('clients', count_connected_clients()))
+
+
+def count_connected_clients():
+    clients = 0
+    
+    # Single connected users looks like this:
+
+    # ---- 8< ----
+    # vpn_by_id_group_xyz: #974, ESTABLISHED, IKEv2, 4242424242424242_i 4242424242424242_r*
+    #   local  'XXXXXX.innogames.de' @ XXX.XXX.XXX.XXX[4500]
```

Comment:
Maybe rather go for `'XXXXX.example.com'` for the example?

Reply:
Fixed.